starknet_devnet/blocks.py

```diff
@@ -77,22 +77,31 @@ class DevnetBlocks:
     def __init__(self, origin: Origin, lite=False) -> None:
         self.origin = origin
         self.lite = lite
-        self.__num2block: Dict[int, StarknetBlock] = {}
+        self.__hash2block: Dict[int, StarknetBlock] = {}
         self.__state_updates: Dict[int, BlockStateUpdate] = {}
-        self.__hash2num: Dict[str, int] = {}
+        self.__num2hash: Dict[int, int] = {}
         self.__pending_block: StarknetBlock = None
         self.__pending_state_update: BlockStateUpdate = None
         self.__pending_signatures: Sequence[List[int]] = None
         self.__state_archive = MemoryStateArchive()
 
     async def get_last_block(self) -> StarknetBlock:
         """Returns the last block stored so far."""
-        number_of_blocks = self.get_number_of_blocks()
+        number_of_blocks = self.get_number_of_not_aborted_blocks()
         return await self.get_by_number(number_of_blocks - 1)
 
     def get_number_of_blocks(self) -> int:
```

Comment:
Why do we need both `get_number_of_blocks` and `get_number_of_not_aborted_blocks`?

I'm worried because rpc/blocks.py is relying on this function and its semantics is being altered.

Reply:
I take a closer look and actually we just need `get_number_of_accepted_blocks()`